src/components/core/Headline.tsx

```diff
@@ -39,7 +39,7 @@ export class HeadlineStateless extends
         return (
             <IntlWrapper locale={locale}>
                 <IntlTranslationsProvider>
-                    {(props: ITranslationsComponentProps) => (
+                    {(props: ITranslationsComponentProps & InjectedIntlProps) => (
```

Comment:
could you please consider this?

Reply:
` & InjectedIntlProps` is not needed. It is some junk code. Will be removed.